src/Entities/ConnectedStuff/ConnectedTempleCrackedBlock.cs

```diff
@@ -0,0 +1,280 @@
+﻿using Celeste.Mod.Entities;
+using Microsoft.Xna.Framework;
+using Mono.Cecil.Cil;
+using Monocle;
+using MonoMod.Cil;
+using MonoMod.RuntimeDetour;
+using MonoMod.Utils;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Celeste.Mod.CommunalHelper.Entities.ConnectedStuff {
```

Comment:
All Communal Helper entities reside in the `Celeste.Mod.CommunalHelper.Entities` namespace.
```suggestion
namespace Celeste.Mod.CommunalHelper.Entities {
```

Reply:
Updated.